lms/static/sass/inline-discussion-rtl.scss

```diff
@@ -0,0 +1,34 @@
+// From lms-main-v1-rtl.scss
+
+// libs and resets *do not edit*
+@import 'bourbon/bourbon'; // lib - bourbon
+@import 'vendor/bi-app/bi-app-rtl'; // set the layout for right to left languages
+@import 'base/variables-rtl';
```

Comment:
@itsjeyd nit: importing rtl variables before main variables seems a bit awkward - it won't allow overriding the var for RTL (if I understand how scss works right). Is it how it is done with other RTL files?